FRC1076-2016-Competition/src/org/usfirst/frc/team1076/robot/gamepad/OperatorInput.java

```diff
@@ -20,10 +21,28 @@ public double armSpeed() {
 	}
 
 	@Override
+	public double armExtendSpeed() {
+		return gamepad.getAxis(GamepadAxis.LeftY);
+	}
+	
+	@Override
 	public double intakeSpeed() {
 		double in = gamepad.getAxis(GamepadAxis.LeftTrigger);
 		double out = gamepad.getAxis(GamepadAxis.RightTrigger);
 		return in - out;
-	}
+	}	
 
+	@Override
+	public IntakeRaiseState intakeRaiseState() {
+		boolean up = gamepad.getButton(GamepadButton.LB);
+		boolean down = gamepad.getButton(GamepadButton.RB);
+		if (up && down) {

```

Comment:
On the gamepad, is Neutral intended to be used when both left and right buttons are pressed as well as when neither are pressed?